app/views/companies/index.html.erb

```diff
@@ -3,25 +3,26 @@
   <table class="table">
     <thead>
       <tr>
-      <th scope="row" >Name</th>
-      <th scope="row" >Address</th>
-      <th scope="row" >Id</th> 
-      <th scope="row" ></th> 
-      <th scope="row" ></th>  
+        <th scope="row" >Name</th>
+        <th scope="row" >Address</th>
+        <th scope="row" >Id</th> 
+        <th scope="row" ></th> 
+        <th scope="row" ></th>  
```

Comment:
What is the propuse of thi two `<th>` labels?

Reply:
It's a space for the edit and delete buttons